docker-compose.yml

```diff
@@ -11,8 +11,12 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock
     entrypoint: /beacon-kit/testing/files/entrypoint.sh
     depends_on:
-      - celestia-network-bridge
-      - reth
+      celestia-network-validator:
+        condition: service_healthy
+      celestia-network-bridge:
+        condition: service_started
+      reth:
+        condition: service_started
```

Comment:
I don't think this makes any logical difference as the `celestia-network-bridge` doesn't start until the validator is "healthy" anyway. 

If you wanted to add a health check to the bridge node, you could try using some lightweight method like `/network_head`.

Reply:
Good point. I tried to add a health check using node.Info or p2p.Info but both required an admin token. Looks like `/network_head` only needs a read auth token so seems a tad easier.

Ref: https://node-rpc-docs.celestia.org/?version=v0.20.4 

Created https://github.com/celestiaorg/celestia-node/issues/4007 so celestia-node exposes an un authenticated status endpoint. In the meantime DevOps shared a workaround that they use:

```shell
exec [sh -c AUTH_TOKEN=$(celestia bridge auth read --node.store=/home/celestia 2>/dev/null); curl --max-time 0.5 -s -X POST -H "Content-type: application/json" -H "Accept: application/json" -H "Authorization: Bearer $AUTH_TOKEN" -d '{"id":1,"jsonrpc":"2.0","method":"header.SyncWait","params":[]}' "http://localhost:26658"; if [ "$?" -eq 0 ]; then exit 0; else echo "Catching up"; exit 1; fi ]
```

so I'll try something like that